### ISSUE TITLE
output by cost instead of # errors

### ISSUE THREAD

**Issue by compbio** [Wikinaut/agrep#9]
Hi,

I am a post-doc in Stanford University, who is bioinformatician on cancer genomic field.
Thank you for making great program.
I am just wondering I can select records that have at most # cost instead of # errors.
This option enables me to differentiate between substitution and insertion/deletion.
It would help my research a lot!!

**Comment by Wikinaut** [comment]
Have you checked the numerous help pages http://www.tgries.de/agrep/agrephlp.html ?

**Comment by compbio** [comment]
Yes, I did and tried every options available. I still could not figure out.
